mailbox/api/pom.xml

```diff
@@ -93,6 +93,10 @@
             <artifactId>assertj-guava</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>java-hamcrest</artifactId>
```

Comment:
Shouldn't it be in test scope?

Reply:
I updated https://github.com/linagora/james-project/pull/1631/commits/61eba12f45bcc92d1fa28fd2a53b13b8e0065673